project/Build.scala

```diff
@@ -63,9 +63,9 @@ object DottyJSPlugin extends AutoPlugin {
 }
 
 object Build {
-  val referenceVersion = "3.0.0"
+  val referenceVersion = "3.0.1-RC1"
 
-  val baseVersion = "3.0.1-RC1"
+  val baseVersion = "3.0.2-RC1"
```

Comment:
Shouldn't the base version be `3.0.1-RC2` for now?

Then once `3.0.1` is release we would change this to `3.0.2-RC1` and forget about the `3.0.1-RC2`. It is in case we need an `RC2`, we should not need to jump to a newer older version (this has caused confusion in the past).

Reply:
> Shouldn't the base version be 3.0.1-RC2 for now?

No, the master branch should be 3.0.2-... now, if there's a 3.0.1-RC2 it should happen in the release-3.0.1 branch (where we should only backport critical fixes): https://github.com/lampepfl/dotty/tree/release-3.0.1